images/win/scripts/Installers/Install-CommonUtils.ps1

```diff
@@ -0,0 +1,13 @@
+$commonPackages = (Get-ToolsetContent).choco.common_packages
+
+foreach ($package in $commonPackages)
+{
+    Write-Host "Installing ${package.name}"
```

Comment:
I don't think this is necessary, choco outputs the package name it is currently installing.